controllers/books.js

```diff
@@ -4,42 +4,48 @@ let books = []
 
 
 export const getBooks = (req, res) => {
-  res.send(books);
-}
+      res.send(books);
+   }
```

Comment:
Still not well indented - Lines 7 & 8.
Set your text editor to use 4 spaces for tab spacing.